docs/flutter/meeting-locale.mdx

```diff
@@ -31,68 +31,84 @@ Navigator.push(context, MaterialPageRoute(builder: (context) => uikit),);
+| "questionAndOptionsCantBeEmpty"                                   | "_\<locale-word>_"   |
+| "polls"                                                           | "_\<locale-word>_"   |
+| "pollBy"                                                          | "_\<locale-word>_"   |
+| "vote"                                                            | "_\<locale-word>_"   |
+| "voted"                                                           | "_\<locale-word>_"   |
+| "viewVoters"                                                      | "_\<locale-word>_"   |
+| "stopRecording"                                                   | "_\<locale-word>_"   |
+| "startRecording"                                                  | "_\<locale-word>_"   |
+| "muteAll"                                                         | "_\<locale-word>_"   |
+| "disableAllVideos"                                                | "_\<locale-word>_"   |
+| "settings"                                                        | "_\<locale-word>_"   |
+| "rec"                                                             | "_\<locale-word>_"   |
+| "camera"                                                          | "_\<locale-word>_"   |
+| "microphoneInput"                                                 | "_\<locale-word>_"   |
+| "chat"                                                            | "_\<locale-word>_"   |
+| "noMessages"                                                      | "_\<locale-word>_"   |
+| "chatMessagesWillAppearHere"                                      | "_\<locale-word>_"   |
+| "file"                                                            | "_\<locale-word>_"   |
+| "image"                                                           | "_\<locale-word>_"   |
+| "send"                                                            | "_\<locale-word>_"   |
+| "participants"                                                    | "_\<locale-word>_"   |
+| "waitlisted"                                                      | "_\<locale-word>_"   |
+| "inCall"                                                          | "_\<locale-word>_"   |
+| "you"                                                             | "_\<locale-word>_"   |
+| "turnOffVideo"                                                    | "_\<locale-word>_"   |
+| "videoAlreadyOff"                                                 | "_\<locale-word>_"   |
+| "dyte"                                                            | "_\<locale-word>_"   |
+| "back"                                                            | "_\<locale-word>_"   |
+| "waitingForTheHostToLetYouIn"                                     | "_\<locale-word>_"   |
+| "shareScreen"                                                     | "_\<locale-word>_"   |
+| "stopSharing"                                                     | "_\<locale-word>_"   |
 
 3. All set, run the app and view the UI Kit in the desired language!
```

Comment:
(suggestion): The 3rd point feels a bit out-of-flow due to the big table between the 2nd & 3rd point. 
How about moving the 3rd point above and keeping the "strings table" under a separate header at the bottom of the page?

Reply:
Sure, we can do that, it justs mentions that to run the app. Don't think it'll make much of a difference.